app/app.ts

```diff
@@ -25,6 +25,8 @@ export class ChemiscopeApp {
     /// CSS style sheet to hide the setting in loading panel about on-demand
     /// loading
     private hideOnDemandStructures: HTMLStyleElement;
+    private warnings: Warnings;
```

Comment:
Naming-wise, I'd rather call this 
```suggestion
    private warningHandler: WarningHandlers;
```

since this not about the warning themselves, but about what to do with them